lib/datapacktypes/omniscript.js

```diff
@@ -87,36 +87,57 @@ OmniScript.prototype.createTitleObjects = function(input) {
+    }
+}
+
 OmniScript.prototype.afterActivationSuccess = async function(inputMap) {
     var jobInfo = inputMap.jobInfo;
     var dataPack = inputMap.dataPack;
+    let omniScriptId = jobInfo.sourceKeyToRecordId[dataPack.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0].VlocityRecordSourceKey];
 
-    if(!jobInfo.ignoreLWCActivation && !jobInfo.checkLWCActivation){
-        var minVersions = yaml.safeLoad(fs.readFileSync(path.join(__dirname,'..', 'buildToolsMinVersionOS-LWC.yaml'), 'utf8'));
-        var minVersion = minVersions[this.vlocity.namespace]
-        var currentVersion = this.vlocity.PackageVersion;
-        if(minVersion && currentVersion && !(currentVersion >= minVersion)){
-            jobInfo.ignoreLWCActivation = true;
-            VlocityUtils.error('LWC Activation', 'Unsupported Managed package Version, LWC Activation disabled');
-        }
-        jobInfo.checkLWCActivation = true;
-    }
+    if (dataPack.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__IsLwcEnabled__c']) {
 
-    if (jobInfo.isRetry && jobInfo.omniScriptLwcActivationSkip[dataPack.VlocityDataPackKey]) { 
-        var oldError = jobInfo.omniScriptLwcActivationSkip[dataPack.VlocityDataPackKey];
-        jobInfo.hasError = true;
-        jobInfo.currentStatus[dataPack.VlocityDataPackKey] = 'Error';
-        jobInfo.currentErrors[dataPack.VlocityDataPackKey] = 'LWC Activation Error >> ' + dataPack.VlocityDataPackKey + oldError;
-        jobInfo.errors.push('LWC Activation Error >> ' + dataPack.VlocityDataPackKey + oldError);
-        VlocityUtils.error('LWC Activation Error', dataPack.VlocityDataPackKey + oldError);
-        return;
-    }
+        if(!jobInfo.ignoreLWCActivation && !jobInfo.checkLWCActivation){
+            var minVersions = yaml.safeLoad(fs.readFileSync(path.join(__dirname,'..', 'buildToolsMinVersionOS-LWC.yaml'), 'utf8'));
+            var minVersion = minVersions[this.vlocity.namespace]
+            var currentVersion = this.vlocity.PackageVersion;
+            if(minVersion && currentVersion && !(currentVersion >= minVersion)){
+                jobInfo.ignoreLWCActivation = true;
+                VlocityUtils.error('LWC Activation', 'Unsupported Managed package Version, LWC Activation disabled');
+            }
+            jobInfo.checkLWCActivation = true;
+        }
 
-    if (jobInfo.ignoreLWCActivation) {
-        return;
-    }
+        if (jobInfo.isRetry && jobInfo.omniScriptLwcActivationSkip[dataPack.VlocityDataPackKey]) { 
+            var oldError = jobInfo.omniScriptLwcActivationSkip[dataPack.VlocityDataPackKey];
+            jobInfo.hasError = true;
+            jobInfo.currentStatus[dataPack.VlocityDataPackKey] = 'Error';
+            jobInfo.currentErrors[dataPack.VlocityDataPackKey] = 'LWC Activation Error >> ' + dataPack.VlocityDataPackKey + oldError;
```

Comment:
Probably need a space here in the error message?

Reply:
Agree.. Done